contribs/gmf/apps/desktop/index.html

```diff
@@ -99,6 +99,17 @@
           gmf-search-coordinatesprojections="mainCtrl.searchCoordinatesProjections"
           gmf-search-clearbutton="true">
         </gmf-search>
+        <div class="gmf-backgroundlayerbutton btn-group dropup">
+          <button
+              class="btn btn-default"
+              data-toggle="dropdown">
+            <img src="https://cloud.githubusercontent.com/assets/319774/13696817/1cc9a7ee-e769-11e5-85c8-a1ea5ff21a5d.png" alt="" />
```

Comment:
Please, see with @pgiraud or @fredj .  I don't know the answer to that question.